cajun-jsonapi.spec

```diff
@@ -34,17 +35,19 @@ and design.
 %build
 
 %install
-install -d -m755 $RPM_BUILD_ROOT/%{_includedir}/cajun/json
-install -p -m644 json/* $RPM_BUILD_ROOT/%{_includedir}/cajun/json
+install -d -m755 %{buildroot}%{_includedir}/cajun/json
+install -p -m644 json/* %{buildroot}%{_includedir}/cajun/json
 
 %check
 make %{?_smp_mflags}
 
+%clean
+rm -rf %{buildroot}
+
 %files devel
+%defattr(-, root, root)
 %doc Readme.txt ReleaseNotes.txt
-%dir %{_includedir}/cajun
```

Comment:
I wouldn't expect it because if a directory is specified like
%{_includedir}/cajun/ 
the %dir is implicitely included.
In cases where something is ../cajun/* it wouldn't and an additional %dir would be required.

I _do not think_ that Fedora RPM is different in that regard.